heartbeat/ip-jail

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env bash
```

Comment:
We prefer /bin/sh. If you only need bash for =~ below you could replace it with case and *:* (example: https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/findif.sh#L234-L241).

If not there's the "@BASH_SHELL@" variable you could use, and rename it to ip-jail.in and add it to configure.ac so it gets updated when you run ./configure.